tools/tslint-rules/noUnscapedHtmlTagRule.js

```diff
@@ -0,0 +1,64 @@
+const ts = require('typescript');
+const utils = require('tsutils');
+const Lint = require('tslint');
+
+const ERROR_MESSAGE =
+    'A HTML tag may only appear if it is escaped. ' +
+    'This is meant to prevent failures in docs generation caused by a misinterpreted tag.';
+
+/**
+ * Rule that walks through all comments inside of the library and adds failures when it
+ * detects unescaped HTML tags inside of multi-line comments.
+ */
+class Rule extends Lint.Rules.AbstractRule {
+
+  apply(sourceFile) {
+    return this.applyWithWalker(new NoUnescapedHtmlTagWalker(sourceFile, this.getOptions()));
+  }
+}
+
+class NoUnescapedHtmlTagWalker extends Lint.RuleWalker {
+
+  visitSourceFile(sourceFile) {
+    utils.forEachComment(sourceFile, (fullText, commentRange) => {
+
+      let isEscapedHtmlTag = true;
+      while (true) {
```

Comment:
I think this would be much easier with a regex. A simple one like this would be effective for when either character is unescaped,

```
/[^`][<>]/g
```

You could certainly extend it to look for unmatched pairs, but I'm not sure that's even necessary.

Reply:
When I closed this I was just thinking it could be better to just look for unmatched pairs. In fact it was my initial thought. Than I think I forget about it and start looking for the &lt; and &gt; matching pairs. I'll review and reopen it, probably tomorow.